examples/notebooks/CycleGAN.ipynb

```diff
@@ -69,6 +69,37 @@
     "```"
    ]
   },
+  {
```

Comment:
I think here we miss pytorch-ignite installation command for Colab ?

Reply:
fixed. thanks!